### PR TITLE
Add support for Kafka Topic User ACL management

### PR DESCRIPTION
**Pull request by dweinshenker**
**What**

- Add support for kafka topic user acls based on godo updates (https://github.com/digitalocean/godo/pull/633)

**Testing**

Ran`TestAccDigitalOceanDatabaseUser_KafkaACLs,TestAccDigitalOceanDatabaseUser_Basic,TestAccDigitalOceanDatabaseUser_MongoDB,TestAccDigitalOceanDatabaseUser_MongoDBMultiUser,TestAccDigitalOceanDatabaseUser_MySQLAuth` acceptance tests

### CODE REVIEW

**Review by andrewsomething — APPROVED** [review]
:+1:  Looks great! Sorry for the delayed review.

File: digitalocean/database/resource_database_user.go [hunk 1]
```diff
@@ -67,6 +78,32 @@ func ResourceDigitalOceanDatabaseUser() *schema.Resource {
...
+				Computed: true,
+			},
+			"topic": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"permission": {
+				Type:     schema.TypeString,
+				Required: true,
```
Comment (andrewsomething) [inline digitalocean/database/resource_database_user.go:95]: On second look, I'm realizing that updates are not actually supported. Combined with the fact that we're not setting state on read as `GetUser` responses not including `settings`, we mislead users into thinking that an update is successfully applied.  So we should add `ForceNew` to these to make it clear that in-place updates are not supported. 

```suggestion
				Type:         schema.TypeString,
				Required:     true,
				ForceNew:     true,
				ValidateFunc: validation.NoZeroValues,
			},
			"permission": {
				Type:     schema.TypeString,
				Required: true,
				ForceNew: true,
```

**Review by andrewsomething — CHANGES_REQUESTED** [review]

File: digitalocean/database/resource_database_user.go (hunk 1 above)
Comment (dweinshenker) [reply digitalocean/database/resource_database_user.go]: I agree with your suggestion, that makes it more clear that these changes are not allowed in-place. Thanks for the review @andrewsomething !

**Review by andrewsomething — APPROVED** [review]
:+1:

**Pull request merged**